force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +409,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        Integer myWord = -1;
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        for (Integer i = 0; i < words.size(); i++) {
+            if (words [i] == wordToFind) {
+            myWord = i;
+            break;
+            }
+            
+        }
+        return myWord; // Replace null with the variable you used to store the result
```

Comment:
God job breaking when you found the match.  Another technique is to just return. Both are fine. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
 ```